### PR TITLE
Add monthly update for September 2022

### PR DESCRIPTION
**Pull request by mbasmanova**

### CODE REVIEW

**Review by xiaoxmeng — APPROVED** [review]
LGTM. Thanks!

File: velox/docs/monthly-updates/september-2022.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+* Add support for :ref:`flattening concat-like function calls<expr-flatten-concat>`
+  in expression evaluation.
+
+Presto Functions
+================
+
+* Add :func:`rank`, :func:`dense_rank` and :func:`percent_rank` window functions.
+* Add :func:`zip_with` function.
+* Add support for casting JSON string to a struct.
+* Add support for decimal inputs to :func:`multiply`, :func:`divide`,
```
Comment (majetideepak) [inline velox/docs/monthly-updates/september-2022.rst:27]: `decimal` -> `DECIMAL`?

**Review by majetideepak — APPROVED** [review]
@mbasmanova thanks for putting this together.
Just a comment.

File: velox/docs/monthly-updates/september-2022.rst (hunk 1 above)
Comment (mbasmanova) [reply velox/docs/monthly-updates/september-2022.rst]: Thanks. Updated.

File: velox/docs/monthly-updates/september-2022.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  system exceptions only. :pr:`2662`
+* Fix multiple issues found by the `ThreadSanitizer <https://clang.llvm.org/docs/ThreadSanitizer.html>`_.
+
+Credits
+=======
+
+Aditi Pandit, Austin Dickey, Behnam Robatmili, Bikramjeet Vig, Bo Yang, Deepak
+Majeti, Huameng Jiang, Jake Jung, Jialiang Tan, Jimmy Lu, Karteek Murthy Samba
+Murthy, Kevin Wilfong, Krishna Pai, Laith Sakka, Masha Basmanova, Michael
+Bolin, Miłosz Linkiewicz, Orri Erling, Pavel Solodovnikov, Pedro Eugenio Rocha
```
Comment (aditi-pandit) [inline velox/docs/monthly-updates/september-2022.rst:60]: Pedro's name is repeated twice.

**Review by aditi-pandit — APPROVED** [review]
Thanks Masha.

File: velox/docs/monthly-updates/september-2022.rst (hunk 2 above)
Comment (mbasmanova) [reply velox/docs/monthly-updates/september-2022.rst:60]: Good catch. Removed.

**Pull request closed without merging**